capstone/src/main/webapp/script.js

```diff
@@ -327,24 +317,30 @@ function isStringEmpty(str) {
 }
 
 /** post request params to send a POST request using fetch() */
-const requestParamPOST = {
-  method: 'POST',
-  headers: {
-    'Content-Type': 'application/json',
-  },
-};
+// const requestParamPOST = {
```

Comment:
Sorry if I missed this before, but why are we commenting this part out?

Reply:
+1